pyproject.toml

```diff
@@ -19,9 +19,33 @@ dependencies = [
     "iso8601",
 ]
 
+[project.optional-dependencies]
+dev = [
+    "build",
+    "coverage",
+    "ipython",
+    "isort",
```

Comment:
isort is not used yet IIRC

Reply:
Well, it *is* optional after all.  Chances are we'll use ruff for this too at some point...